fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -20,7 +20,7 @@ object CardStyle {
   val news = "news"
 
   def apply(content: Content, trailMetaData: MetaDataCommonFields): CardStyle = {
-    val href = trailMetaData.href.orElse(content.safeFields.get("href"))
```

Comment:
I checked with @DiegoVazquezNanini and @janua, and we don't think that snap href will never be in content api.